view.py

```diff
@@ -0,0 +1,78 @@
+def greetings():
+    print('Добро пожаловать в телефонный справочник!')
+
+def menu():
+    print('Пожалуйста выберите команду для выполнения:\n',
+          '1 - показать все контакты\n',
+          '2 - добавление контакта\n',
+          '3 - поиск контакта\n',
+          '4 - изменение контакта\n',
+          '5 - удаление контакта\n',
+          '6 - выход из справочника')
+    answer = int(input('Введите номер команды: '))
+    return answer
+
+def show_contacts(data):
+    if data == 'None':
+        print('Справочник пуст')
+    else: print(data)
```

Comment:
Можно было бы чуточку по красивее выводить контакта.  
![image](https://github.com/TanyaKonyukhova/GB_lesson_8/assets/49741194/4dae29a4-676b-4790-96b2-a1320ae7c16c)
Хотя бы так
```suggestion
    else:
        print(*data, sep='\n')
```